src/ImageSharp/Formats/Bmp/BmpInfoHeader.cs

```diff
@@ -545,13 +555,19 @@ public void WriteV5Header(Span<byte> buffer)
 
         internal void VerifyDimensions()
         {
-            const int MaximumBmpDimension = 65535;
-
             if (this.Width > MaximumBmpDimension || this.Height > MaximumBmpDimension)
```

Comment:
I dont think this check makes sense anymore since we are checking against size > 2147483647 and the width and height of an image is `int` and therefore cannot be greater then this. I think we should remove this `VerifyDimensions` now.

Reply:
removed with 1acbfbe3